R/rdeephaven/R/table_ops.R

```diff
@@ -0,0 +1,643 @@
+setGeneric(
+  "merge",
+  function(x, y = NULL, ...) {
+    standardGeneric("merge")
+  }
+)
+
+# We need many implementations of 'merge' because of the kind of args it should support
+
+base_merge <- function(x, y, ...) {
+  arg_list <- c(unlist(x), unlist(y), unlist(c(...)))
+  if (length(arg_list) == 1) {
+    return(arg_list[[1]])
+  }
+  unwrapped_arg_list <- lapply(arg_list, strip_s4_wrapping)
+  return(new("TableHandle", .internal_rcpp_object = unwrapped_arg_list[[1]]$merge(unwrapped_arg_list[2:length(unwrapped_arg_list)])))
```

Comment:
I don't think that nulls in the varargs get stripped out before calling merge.

Reply:
```
> unlist(c(NULL, 1, 2, 3, NULL, 4, NULL, NULL, 5))
[1] 1 2 3 4 5
```

The call to `unlist(c(...))` in `arg_list <- c(unlist(x), unlist(y), unlist(c(...)))` takes care of it by default.